CucumberTest/application.properties

```diff
@@ -0,0 +1 @@
+Path=/E://selenium//chromedriver_win32//chromedriver.exe/

```

Comment:
The path is as per the local system. Please make it dynamic.

Reply:
okay

Reply:
driver will go in resources source dir